### PR TITLE
[14.0] [MIG] sale_report_hide_price: Module migrated in v14.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
- Module developed by **Pragati Patel**. (Assigned by **Yogesh Mahera**)

### CODE REVIEW

**Comment by Chandresh-SerpentCS** [comment]
@YogeshMahera-SerpentCS Please review the PR. Thanks.

File: sale_report_hide_price/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2004-2010 OpenERP SA (<http://www.openerp.com>)
+    Copyright (C) 2011-Today Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
```
Comment (JayVora-SerpentCS) [inline sale_report_hide_price/LICENSE/LICENSE]: 2020-Today

File: sale_report_hide_price/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "Show Price",
+        help="If checked, you can see the price in "
+        "report of"
+        "Sales Order / Quotation.",
+    )
+    show_discount = fields.Boolean(
+        "Show Discount",
+        help="If checked, you can see the discount"
+        " in report of Sales Order/Quotation.",
+    )
```
Comment (JayVora-SerpentCS) [inline sale_report_hide_price/models/sale.py:21]: @ForamSerpentCS We should have this feature linked with Security groups.
These fields should be seen by ACCOUNT Manager (or top account group).

Comment (ForamSerpentCS) [reply sale_report_hide_price/models/sale.py:21]: Yes we may, If we can have security groups for it, then we can manage this feature directly applying field security in template.

**Comment by pragati-patel98** [comment]
@ForamSerpentCS please review the code.

**Review by ForamSerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
@ForamSerpentCS could you test?

**Comment by ForamSerpentCS** [comment]
> @ForamSerpentCS could you test?

It contains syntax error. Will make it updated.
@pragati-patel98

**Comment by JayVora-SerpentCS** [comment]
Let me know when its ready

**Comment by ForamSerpentCS** [comment]
> Let me know when its ready

Have tested it, It's now ready to merge.

**Pull request merged**